ops/terraform/modules/resources/eft_efs/variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "partner" {
+  description = "Parnert name. E.g., 'bcda', 'dpc', etc."
+  type        = string
+}
+
+variable "partner_acct_num" {
+  description = "Partners AWS Account Number"
+  type        = string
+}
+
+variable "partner_subnets" {
+  description = "Partner subnets requiring access to EFT EFS file systems"
+  type        = map(list(string))
+}
+
+variable "posix_uid" {
+  description = "High numbered posix id for tracking partner owned files. E.g., 1500, 1501, etc"
+}
+
+variable "posix_gid" {
+  description = "High numbered posix id for tracking partner owned files. E.g., 1500, 1501, etc"
+}
```

Comment:
Do these values matter or are they arbitrary?

Reply:
It does not matter as long as they do not fall in any reserved id ranges (varies by os). Usually this is 0-999 for service accounts and real users. Systemd also reserves really high 6000+, so I'm suggesting 1500 to be safe. 

It really does not matter in the end as long as the uid/gid is not in use by either BFD or the partner and we agree to these id's before hand.